mockative/src/commonMain/kotlin/io/mockative/ResultBuilder.kt

```diff
@@ -12,7 +12,13 @@ class ResultBuilder<R>(
         mock.addBlockingStub(ClosedBlockingStub(expectation, blocks))
     }
 
-    override fun invokes(block: () -> R) = invokes { _ -> block() }
+    fun returns(value: R) = invokes { value }
 
-    override fun invokesMany(vararg blocks: () -> R) = invokesMany(blocks.map { block -> { _ -> block() } })
```

Comment:
What is the reasoning behind removing the vararg variant?

1. It's a breaking change in a **patch** version
2. Without the vararg, every use site has to do a `listOf(...)` or create a helper function to polyfill the existing method.

You can see an example failure here:

https://github.com/TWiStErRob/net.twisterrob.travel/pull/207#issuecomment-2132321041

and an example fix:

https://github.com/TWiStErRob/net.twisterrob.travel/pull/207/commits/e294dfa5111ef9cae9ab818b4cd29d737b12caa4

I'm only using this in a single class, imagine a project that fully relies on this...

Reply:
Good point @TWiStErRob! I guess I imagined nobody actually used this function since I personally didn't think it was that useful. I'll add it back in and release a hotfix for it in `v2.2.2` shortly, and be more careful of retaining backward compatibility going forward, making sure to use `@Deprecated` warnings for future occurrences.

Reply:
No worries.

Tip: write tests for these methods and it'll show you when they're useful, and whether the DX of them is good :)